### ISSUE TITLE
Implement .echo directive

### ISSUE THREAD

**Issue by boeckmann** [boeckmann/asm6502#15]
_.echo_ lets the user output strings and the result of numerical expressions to stdout while assembling. It is run at assembler pass 2.

**Comment by boeckmann** [comment]
Implemented by dc4fdc1.

**Comment by boeckmann** [comment]
.echo format specifier `[$]` makes a number being printed in hexadecimal representation.

Example:
```
.echo "hexadecimal representation of ", 4711, " is ", [$]4711
```
results in following output:
```
hexadecimal representation of 4711 is $1267
```
